src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -1329,17 +934,454 @@ type TableContext =
+                                    ?precondition : Expr<'TRecord -> bool>, ?returnLatest : bool) =
+            __.UpdateItemAsync(key, updater, ?precondition = precondition, ?returnLatest = returnLatest)
+            |> Async.RunSynchronously
+
+        /// <summary>
+        ///     Updates item with supplied key using provided record update operation.
+        /// </summary>
+        /// <param name="key">Key of item to be updated.</param>
+        /// <param name="updater">Table update expression.</param>
+        /// <param name="precondition">Specifies a precondition expression that existing item should satisfy.</param>
+        /// <param name="returnLatest">Specifies the operation should return the latest (true) or older (false) version of the item. Defaults to latest.</param>
+        member __.UpdateItem(key : TableKey, updater : Expr<'TRecord -> UpdateOp>,
+                                    ?precondition : Expr<'TRecord -> bool>, ?returnLatest : bool) =
+            __.UpdateItemAsync(key, updater, ?precondition = precondition, ?returnLatest = returnLatest)
+            |> Async.RunSynchronously
+
+
+        /// <summary>
+        ///     Checks whether item of supplied key exists in table.
+        /// </summary>
+        /// <param name="key">Key to be checked.</param>
+        member __.ContainsKey(key : TableKey) =
+            __.ContainsKeyAsync(key) |> Async.RunSynchronously
+
+        /// <summary>
+        ///     Fetches item of given key from table.
+        /// </summary>
+        /// <param name="key">Key of item to be fetched.</param>
+        member __.GetItem(key : TableKey) = __.GetItemAsync(key) |> Async.RunSynchronously
+
+
+        /// <summary>
+        ///     Fetches item of given key from table.
+        ///     Uses supplied projection expression to narrow downloaded attributes.
+        ///     Projection type must be a tuple of zero or more non-conflicting properties.
+        /// </summary>
+        /// <param name="key">Key of item to be fetched.</param>
+        /// <param name="projection">Projection expression to be applied to item.</param>
+        member __.GetItemProjected(key : TableKey, projection : ProjectionExpression<'TRecord, 'TProjection>) : 'TProjection =
+            __.GetItemProjectedAsync(key, projection) |> Async.RunSynchronously
+
+        /// <summary>
+        ///     Fetches item of given key from table.
+        ///     Uses supplied projection expression to narrow downloaded attributes.
+        ///     Projection type must be a tuple of zero or more non-conflicting properties.
+        /// </summary>
+        /// <param name="key">Key of item to be fetched.</param>
+        /// <param name="projection">Projection expression to be applied to item.</param>
+        member __.GetItemProjected(key : TableKey, projection : Expr<'TRecord -> 'TProjection>) : 'TProjection =
+            // TOCONSIDER implement in terms of Async equivalent as per the rest
```

Comment:
(there may be a subtle reason not to do this so leaving it bug-compatible for now)

Reply:
No idea, sorry. Happy to leave as-is.